app/notifications/notifications.component.html

```diff
@@ -2,11 +2,33 @@
   <h1>{{ 'my notifications' | translate }}</h1>
 </md-card>
 <md-card class="default-card">
+  <div class="filter-options" *ngIf="notifications.length != 0">
+    <span class="bg-info info">FILTER BY:</span>
+    <select (change)="selectStatus($event.target.value)">
```

Comment:
Since md-select is out, we should start to use it:
```
<md-select name="items" [(ngModel)]="itemSelected">
    <md-option value="1">item1</md-option>
    <md-option value="2">item2</md-option>
</md-select>
```